pkg/echimpcache/echimp_cache.go

```diff
@@ -0,0 +1,192 @@
+package echimpcache
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/parquet"
+	"github.com/cms-enterprise/mint-app/pkg/s3"
+)
+
+// echimpCacheTimeHours is the length of time before a echimpCache needs to be refreshed
+const echimpCacheTimeHours = 3
+const crKey = "echimp_cr"
+const tdlKey = "echimp_tdl"
```

Comment:
I think that might actually be the case, since it looks like `package echimptestdata` codes in the same thing.

We don't have to address it as part of this PR if you're able to track this as a TODO with a comment -- but I think making this a configurable path is probably in our best interest!

Reply:
Thanks Clay, if you are ok with it, I think I might just leave it as a todo for now. The wiring for this would be a little convoluted with our current reliance on viper to read the env vars. 

Perhaps we can target this with some of the other changes we talked about for fetching configs? We might also consider making the eChimpCacheTime an envar too, in case we want to have varying times in diff environments